status-im/status-legacy
# datasync persistence

Fixes: #8734 
Fixes: #8807

### Testing
- enable datasync exchange a few messages
- login/logout exchange a few messages
- check messages are exchanged succesfully when both peers go offline/online
- Pair devices and update your picture on device B, should update on device A
status: ready
